vep/__init__.py

```diff
@@ -279,15 +281,45 @@ def install_build_tools(self, pip):
             else:
                 pip('install', "%s==%s" % (tool, version), _out=print_line)
 
+    def _pip_requirements_filename(self, path=''):
+        """
+        Returns filename of pip requirements file, & verifies that the file exists.
+
+        :param path: Path to dir to search. Default it current dir.
+        :type path: str
+        :raises VEPackagerError
+        :return: Filename of pip requirements
+        :rtype: str
+        """
+        if self.args.pip_requirements:
+            filenames = list(self.args.pip_requirements)
+        else:
+            filenames = list(DEFAULT_REQUIREMENTS_FILES)
+
+        path_filename = None
+        found = False
+        for filename in filenames:
+            path_filename = filename if not path else os.path.join(path, filename)
```

Comment:
Default `path` to `'.'` and just use `os.path.join(path, filename)` each time?

Reply:
sounds good. changed.